src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php

```diff
@@ -171,6 +171,7 @@ private function createAssets(array $keys): array
         foreach ($keys as $key) {
             $responses[$key] = $this->client->request('POST', 'assets', [
                 'body' => [
+                    'id' => $key,
```

Comment:
be careful. Translation keys are only unique per domain, not globally. so they might note be suitable as loco ids.

Reply:
You are right, so may be we can concatenate the domain as prefix or suffix to the key?